streamer/src/nonblocking/quic.rs

```diff
@@ -137,7 +138,7 @@ async fn setup_connection(
 
         let remote_addr = connection.remote_address();
 
-        let (mut connection_table_l, stake) = {
+        let (connection_table_l, stake) = {
```

Comment:
i think this should rahter be `Option<(Mutexguard, u64)>`? That's because if connection_table_l is None, stake isn't used at all.

Reply:
Yes, that makes sense. I'll push an update to the PR.